code/__DEFINES/~monkestation/holomaps.dm

```diff
@@ -17,6 +17,9 @@
 #define HOLOMAP_ROCK "#66666644"	// Color of mineral walls
 #define HOLOMAP_HOLOFIER "#0096bb"	// Whole map is multiplied by this to give it a green holoish look
 
+#define HOLOMAP_AREACOLOR_SHIELD_1 rgb(0, 119, 255, 64)
```

Comment:
isn't it `rgba()` that takes 4 args?

Reply:
![image](https://github.com/Monkestation/Monkestation2.0/assets/65794972/6b091cb4-a301-4779-84c8-35f30f24c56d)